test/test.baseexternalclient.ts

```diff
@@ -1053,6 +1053,74 @@ describe('BaseExternalAccountClient', () => {
         scopes.forEach(scope => scope.done());
       });
 
+      it('should return credential with no expiry date if STS response does not return one', async () => {
```

Comment:
I think expires_in is always returned for external account credentials, so this test might not be needed/confusing.

Reply:
Yes, I think this usually not needed. However, since I changed logic by setting expire_in to optional,
I want to cover the line expireDate = null in refreshAccessTokenAsync() else case in baseexternalclient.ts.